spring-cloud-gcp-docs/src/main/asciidoc/logging.adoc

```diff
@@ -99,8 +99,7 @@ Your Logback configuration may then look something like this:
 [source, xml]
 ----
 <configuration>
-  <property name="projectId" value="test-project"/>
-  <include resource="org/springframework/cloud/gcp/logging/logback-json-appender.xml" />
+  <include resource="org/springframework/cloud/gcp/autoconfigure/logging/logback-appender.xml" />
```

Comment:
It should be `..logback-json-appender.xml` not `...logback-appender.xml`.

Reply:
@meltsufin PTAL